examples/llm/vllm/operators/vllm.py

```diff
@@ -26,10 +35,53 @@ def __init__(
             default_factory=dict
         ),
         repository: Optional[str] = None,
-        logger: Optional[Any] = None,
+        logger: Optional[logging.Logger] = None,
+    ):
+        self.name = name
+        self.version = version
+        self.request_plane = request_plane
+        self.data_plane = data_plane
+        if logger is None:
+            self.logger = logging.getLogger(__name__)
+        else:
+            self.logger = logger
+
+        self._init_stages(parameters)
+
+    @abc.abstractmethod
+    def _init_stages(
+        self,
+        parameters: Optional[dict[str, str | int | bool | bytes]] = field(
+            default_factory=dict
+        ),
+    ):
+        pass
+
+    @staticmethod
+    def _prepare_inputs(request: RemoteInferenceRequest):
+        inputs, parameters = {}, {}
+        for input_name, input_data in request.inputs.items():
+            local_tensor = input_data.local_tensor
+            numpy_tensor = np.from_dlpack(local_tensor)
+            input_data.__del__()
+            inputs[input_name] = numpy_tensor
```

Comment:
I think can simplify futher to single line as:

```suggestion
            inputs[input_name] = np.from_dlpack(input_data)
```